addons/hw_drivers/iot_handlers/drivers/DisplayDriver_L.py

```diff
@@ -54,6 +71,8 @@ def __init__(self, identifier, device):
             'get_owner': self._action_get_owner,
         })
 
+        self.set_orientation(Orientation.NORMAL)
```

Comment:
Allow touchscreen synchronization with screen orientation when class is instanciated